eclair-core/src/test/scala/fr/acinq/eclair/payment/relay/ChannelRelayerSpec.scala

```diff
@@ -488,14 +488,12 @@ class ChannelRelayerSpec extends ScalaTestWithActorTestKit(ConfigFactory.load("a
 
     // Simulate a chain re-org that changes the shortChannelId:
     channelRelayer ! WrappedShortChannelIdAssigned(ShortChannelIdAssigned(null, channelId_ab, ShortChannelId(42), Some(channelUpdate_ab.shortChannelId)))
-    val channels7 = getOutgoingChannels(true)
-    assert(channels7.isEmpty)
```

Comment:
Shouldn't we keep that and assert that it's now `nonEmpty` since we changed that behavior?

Reply:
Yeah, btw this was removed from #2224, since `ChannelRelayer` isn't listening to `ShortChannelIdAssigned` anymore. But I should have kept a reorg test using the `LocalChannelUpdate`. Will update #2224 directly if that's okay with you.

Reply:
Good point, that's a nice benefit that we won't have to listen to `ShortChannelIdAssigned` anymore here!